.github/workflows/trello.yml

```diff
@@ -2,14 +2,14 @@ name: Link Trello card
 
 on:
   pull_request:
-    types: [ opened ]
+    types: [ opened , edited]
```

Comment:
This is a late review, but when I wrote this I'd excluded `edited` by choice not accident

Every time a PR is edited the action will run and create another link to Trello - I'd decided to avoid this because the code in the action doesn't check the earlier Trello card comments for existing links?

Reply:
Ah - my bad on this, I assumed it'd check first. It seemed like this was to cover the case where a PR was created without a Trello link _then_ one was added.